embulk-core/src/main/java/org/embulk/deps/SelfContainedJarFile.java

```diff
@@ -64,6 +64,7 @@
             this.manifest = null;
             this.innerResources = null;
             this.innerResourcesBinary = null;
+            closeQuiet(jarInputStream);
```

Comment:
If the execution goes into this `catch` block, it means that it failed in constructing an instance of `JarInputStream`, then `jarInputStream` is not assigned. I don't think we need to close it, then?

Reply:
you are right, I missed the place. Modified.